movieApp/index.html

```diff
@@ -4,7 +4,7 @@
       <title>MovieApp</title>
       <meta charset="UTF-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <link rel="stylesheet" href="styles.css">
+      <link rel="stylesheet" href="asset/styles.css">
```

Comment:
assets folder meant to be used only for images, icons or other media content. Styles can be in styles folder, since you can have multiple css files for different pages.